cypress/integration/buy-shirt.spec.ts

```diff
@@ -1,36 +1,40 @@
-
 import { AddressStepPage, LoginPage, MenuContentPage, PaymentStepPage, ProductsListPage, ShippingstepPage, ShopingCartPage } from "../page/index"
 
-const menuContentPage = new MenuContentPage()
-const productsListPage = new ProductsListPage();
-const shopingCartPage = new ShopingCartPage()
-const loginPage = new LoginPage()
-const addressStepPage = new AddressStepPage()
-const shippingstepPage = new ShippingstepPage()
-const paymentStepPage = new PaymentStepPage()
-
 describe("Buy a t-shirt", () => {
 
-    it("then the t-shirt should be bought", () => {
+  const menuContentPage = new MenuContentPage()
+  const productsListPage = new ProductsListPage()
+  const shopingCartPage = new ShopingCartPage()
+  const loginPage = new LoginPage()
+  const addressStepPage = new AddressStepPage()
+  const shippingstepPage = new ShippingstepPage()
+  const paymentStepPage = new PaymentStepPage()
```

Comment:
Please follow a similar approach to the one that is on the cypress/integration/dresses-list.spec.ts file, let's declare the page objects with let, and then initiliaze them on the beforeAll. By doing that you are applying the Arrange of the AAA :)